packages/core/src/renderer/components/kube-object-details/kube-object-detail-items/metrics/pod-details-pod-metrics.injectable.tsx

```diff
@@ -2,22 +2,20 @@
  * Copyright (c) OpenLens Authors. All rights reserved.
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
-import React from "react";
 import { podDetailsMetricsInjectionToken } from "@k8slens/metrics";
 import { getInjectable } from "@ogre-tools/injectable";
-import { ClusterMetricsResourceType } from "../../../../common/cluster-types";
-import metricsDetailsComponentEnabledInjectable from "../../../api/catalog/entity/metrics-details-component-enabled.injectable";
-import { kubeObjectDetailItemInjectionToken } from "./kube-object-detail-item-injection-token";
-
-const Empty = () => <></>;
+import { ClusterMetricsResourceType } from "../../../../../common/cluster-types";
+import metricsDetailsComponentEnabledInjectable from "../../../../api/catalog/entity/metrics-details-component-enabled.injectable";
+import { kubeObjectDetailItemInjectionToken } from "../kube-object-detail-item-injection-token";
+import { EmptyMetrics } from "./empty-metrics";
 
 const podMetricsInjectable = getInjectable({
-  id: "kube-object-pod-metrics-injectable",
+  id: "pod-details-pod-metrics",
   instantiate: (di) => {
     const podMetrics = di.injectMany(podDetailsMetricsInjectionToken);
     const first = podMetrics[0];
 
-    const Component = first ?? Empty;
+    const Component = first?.Component ?? EmptyMetrics;
```

Comment:
Could we instead register `EmptyMetrics` as implementation of `podDetailsMetricsInjectionToken` when there's nothing better around?